app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -95,15 +102,33 @@ module.exports = React.createClass
     if @props.autoFocus is true
       @handleFocus()
 
+  componentWillReceiveProps: (newProps) ->
+    selects = []
+    if newProps.task isnt @props.task
+      # copy the select configs over from the task, to avoid modifying the task.
+      newProps.task.selects.map (select) -> selects.push Object.assign {}, select
+      @setState {selects}, () => @syncAnnotations newProps.annotation.value
+
   componentDidUpdate: (prevProps) ->
     if prevProps.task isnt @props.task and @props.autoFocus is true
       @handleFocus()
 
+  syncAnnotations: (annotations) ->
+    # check each annotation.value and set the corresponding selected option for each select
+    {selects} = @state
+    annotations.map (annotation, i) =>
+      if annotation.option
+        [selected] = @getOptions(i).filter (option) -> option.value is annotation.value
+        selects[i].selected = selected
+      else
+        selects[i].selected = {label: annotation.value, value: annotation.value}
+    @setState {selects}
+
   handleFocus: ->
```

Comment:
once adding a custom answer to a conditional dropdown that doesn't have any options provided, the focus gets stuck.

i.e. in preview Focus Testing Project, Dropdown Workflow, second task, second dropdown (Smaller Numbers, after selecting anything except 100 for Bigger Number) - type any custom answer.
`projects/markb-panoptes/focus-testing-project/classify?reload=1&workflow=2737`